inappmessaging/USERGUIDE.md

```diff
@@ -312,6 +312,13 @@ Documents targeting Product Managers:
 
 ## <a name="changelog"></a> Changelog
 
+### 4.0.0 (in-progress)
```

Comment:
Version should be 4.0.0-pp-v1

Reply:
updated.
is the "-v1" necessary, since we use semver?

Reply:
changed to "4.0.0-pp"